SConstruct

```diff
@@ -363,7 +363,7 @@ if selected_platform in platform_list:
             print("Tools can only be built with targets 'debug' and 'release_debug'.")
             sys.exit(255)
         suffix += ".opt"
-        env.Append(CCFLAGS=['-DNDEBUG'])
+        env.Append(CPPDEFINES=['NDEBUG'])
```

Comment:
That was a `CCFLAGS`, not `CPPFLAGS`. Is there no `CCDEFINES` we can use?

Reply:
CPPDEFINES is defines for cpp, the C preprocessor. Not C++-specific.